src/Symfony/Bridge/PhpUnit/ClassExistsMock.php

```diff
@@ -18,16 +18,29 @@ class ClassExistsMock
 {
     private static $classes = [];
 
+    private static $enums = [];
+
     /**
      * Configures the classes to be checked upon existence.
      *
-     * @param array $classes Mocked class names as keys (case sensitive, without leading root namespace slash) and booleans as values
+     * @param array $classes Mocked class names as keys (case-sensitive, without leading root namespace slash) and booleans as values
      */
     public static function withMockedClasses(array $classes)
     {
         self::$classes = $classes;
     }
 
+    /**
+     * Configures the enums to be checked upon existence.
+     *
+     * @param array $enums Mocked enums names as keys (case-sensitive, without leading root namespace slash) and booleans as values
+     */
+    public static function withMockedEnums(array $enums)
+    {
+        self::$enums = $enums;
+        self::$classes += $enums;
```

Comment:
im not sure i understand withMockedEnums as symbol names are unique (at the time we decided to call symbols classes, as it's a more recognizable term)

https://3v4l.org/fFhXZ

Reply:
i understand we return true for all *_exists functions, when given a specific type of symbol

but that's still the case now with writing to $classes AND $enums :thinking: 

if we make this distinction i'd prefer to do it all for all symbol types